app/models/manageiq/providers/embedded_ansible/crud_common.rb

```diff
@@ -0,0 +1,112 @@
+module ManageIQ::Providers::EmbeddedAnsible::CrudCommon
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def queue(zone, instance_id, method_name, args, action, auth_user)
+      task_opts = {
+        :action => action,
+        :userid => auth_user || "system"
+      }
+
+      queue_opts = {
+        :args        => args,
+        :priority    => MiqQueue::HIGH_PRIORITY,
```

Comment:
Note:  In one of my commits, I bumped this up (the `[FIXUP] configuration_script_spec.rb` one) to match what was previously done.  This can change, but at the time I was trying to true things up to how they were done before.

Reply:
I think there are actually bugs with that...HIGH_PRIORITY is really meant for UI-based tasks, and little else.  @jrafanie and @bdunne were telling me the other day about a HIGH_PRIORITY race with the UI and I think these methods.

Reply:
That being said...I'm fine with it for now.  We should just reevaluate those later.